mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -148,13 +148,27 @@ def fill_cidrs_from_directional_traffic_rules(self, data, indexed_policy_name, d
                 else:
                     data["cidrs_map_no_except"][indexed_policy_name].append(rule_item["ipBlock"]["cidr"])
             elif "namespaceSelector" in rule_item and "podSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
-            elif "namespaceSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
-            elif "podSelector" in rule_item:
+                namespaces = list_namespaces_by_labels(rule_item["namespaceSelector"]["matchLabels"])
```

Comment:
This should not be called in workflow. This should be part of the object class https://github.com/CentaurusInfra/mizar/blob/dev-next/mizar/obj/networkpolicy.py

Reply:
workflow runs inside control plane. If control plane is in a separate cluster, this will not work. To get all the namespaces, you will need get the networkpolicy object first, and call api using obj.obj_api.list_pod_for_all_namespaces()